lib/DDG/Spice/Goodreads.pm

```diff
@@ -51,6 +51,7 @@ handle query_clean => sub {
           : "title";
     $_ =~ s/$book_pattern//g;       # removing the book query pattern from query
     $_ =~ s/$goodreads_pattern//g;  # removing the goodreads trigger pattern from query
+    $_ =~ s/^\s+|\s+$//g;           # trim spaces from queries
```

Comment:
The query looks much better now without escape characters for spaces that may occur at times in some form.
Earlier: `/js/spice/goodreads/authors-%20kafka`
Now: `/js/spice/goodreads/authors-kafka`